pkg/utils/nodeclaim/nodeclaim.go

```diff
@@ -468,20 +492,3 @@ func OwnerName(obj interface{ GetLabels() map[string]string }) string {
 	}
 	return ""
 }
-
-func IsExpired(obj client.Object, clock clock.Clock, nodePool *v1beta1.NodePool) bool {
-	return clock.Now().After(GetExpirationTime(obj, nodePool))
-}
-
-func GetExpirationTime(obj client.Object, nodePool *v1beta1.NodePool) time.Time {
-	if nodePool == nil || nodePool.Spec.Deprovisioning.ExpirationTTL.Duration >= 0 || obj == nil {
-		// If not defined, return some much larger time.
-		return time.Date(5000, 0, 0, 0, 0, 0, 0, time.UTC)
-	}
-	return obj.GetCreationTimestamp().Add(nodePool.Spec.Deprovisioning.ExpirationTTL.Duration)
-}
-
-func IsPastEmptinessTTL(nodeClaim *v1beta1.NodeClaim, clock clock.Clock, nodePool *v1beta1.NodePool) bool {
-	return nodeClaim.StatusConditions().GetCondition(v1beta1.NodeEmpty).IsTrue() &&
-		!clock.Now().Before(nodeClaim.StatusConditions().GetCondition(v1beta1.NodeEmpty).LastTransitionTime.Inner.Add(nodePool.Spec.Deprovisioning.EmptinessTTL.Duration))
```

Comment:
If it's removed from the nodeclaim and machine utils are we fully removing? Or is it somewhere else?

Reply:
Yes, I'll fully remove this.